src/main/java/org/springframework/data/elasticsearch/client/reactive/DefaultReactiveElasticsearchClient.java

```diff
@@ -653,6 +653,15 @@ private RequestBodySpec sendRequest(WebClient webClient, String logId, Request r
 				.next();
 	}
 
+	@Override
+	public Mono<Boolean> createIndex(HttpHeaders headers,
```

Comment:
this one should have the imported class like in the interface, and the other one in line 649 the fully qualified name